src/components/missions/Mission.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable  camelcase */
```

Comment:
Hi @adamilare

Thanks for your comment. I quite agree with you that situations arise that disabling some check(s) will be the way out. However, when it's avoidable it's better not to.

In this case, the data from the API could be renamed to conform to JavaScript best practices since it's possible that the API is written using a language like the Python (where snake case is the standard for naming variables).